Vagrantfile

```diff
@@ -0,0 +1,15 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "fedora/32-cloud-base"
+  config.vm.provider "virtualbox" do |v|
+  	v.memory = 2048
+  	v.cpus = 2
+  end
+  config.vm.synced_folder "init_scripts/", "/init_scripts", create: false
+  config.vm.synced_folder "selotape/", "/home/vagrant/selotape", create: true
+  config.vm.provision "shell", privileged: true,
+    inline: "/bin/sh /init_scripts/init-root.sh"
```

Comment:
You might want to consider using the path attribute

Reply:
Don't forget to add a comment or/and emoji (thumb up) in case you have decided to follow the review comment.